### DISCUSSION TITLE
How to re-obtain/retreive the Nextcloud AIO passphrase?

### DISCUSSION THREAD

**Discussion by szaimen** [nextcloud/all-in-one#1786] [Wiki]
If you have somehow lost the passphrase that is used for the AIO interface, you can reobtain it by running `sudo docker exec nextcloud-aio-mastercontainer grep password /mnt/docker-aio-config/data/configuration.json`.

**Comment by Sx3** [comment]
Thank you !

**Comment by shardycharacter** [comment]
Thank you!!!

**Comment by ArmanTheParman** [comment]
Didn't work for me. 
I investigated the directory and found that deleting the metadata file resets the password

sudo rm /var/lib/docker/volumes/metadata.db

Initially, when testing, I ignored the passphrase/password. Then when ready to use, reinstalling didn't create a new password, it only just asks for the password.

Deleting that file refreshes it.

**Comment by Jhongt796** [comment]
Really thanks you!!!

**Comment by logichunt319** [comment]
Lifesaver. Thank you

**Comment by richardcrbb** [comment]
I restarted my portainer container 1000 times until I decided to get help from google and found this, Thanks!!!

**Comment by pencit** [comment]
> Replies

Hello, it doesn't work for me.. there was no "password" in configuration.json after do borg backup

**Comment by one3erver** [comment]
Thanks
default password:
> REMOVED

**Reply by szaimen** [reply]
@one3erver this is not a default password but a generated one specific to your instance.

**Reply by szaimen** [reply]
You should change the password now!

**Reply by one3erver** [reply]
@szaimen 
But the interesting thing is that I didn't have the password and I searched for it on the internet.
How can my unique password be on the internet?

**Comment by Tiko-Lz1e** [comment]
I use docker compose and there is no configuration.json in /mnt/docker-aio-config/data, it is just empty
